pkg/controller/smart-clone-controller.go

```diff
@@ -132,7 +132,7 @@ func shouldReconcileSnapshot(snapshot *csiv1.VolumeSnapshot) bool {
 }
 
 func shouldReconcilePvc(pvc *corev1.PersistentVolumeClaim) bool {
-	if pvc.Status.Phase != corev1.ClaimBound {
+	if pvc.Status.Phase == corev1.ClaimLost {
```

Comment:
Correct me if I'm wrong, but the code will now delete the `VolumeSnapshot` when the PVC is pending?  I think that is technically okay because it has a finalizer.  But I'm not sure.  May not want to depend on that.  Maybe it would be better to keep it around until the PVC is bound?

Reply:
So we have 3 possible values for the PVC state, Pending, Bound, and Lost. If due to WaitForFirstConsumer we have to handle pending, and we handle bound, then what are our options?